src/view/TextAreaView.java

```diff
@@ -89,9 +100,14 @@ public void propertyChange(PropertyChangeEvent evt) {
     public void actionPerformed(ActionEvent evt) {
 
     }
-    private static void handleRecommendClick(String option, String text) {
+    private void handleRecommendClick(String option, String text) {
```

Comment:
I believe the controller for recommend (RecommendController) should be responsible for "handleRecommendClick". Something like the following:

RecommendController.execute()....